gunicorn/glogging.py

```diff
@@ -198,6 +198,7 @@ def setup(self, cfg):
         self.loglevel = self.LOG_LEVELS.get(cfg.loglevel.lower(), logging.INFO)
         self.error_log.setLevel(self.loglevel)
         self.access_log.setLevel(logging.INFO)
+        self.datefmt = cfg.logconfig_dict.get('datefmt', self.default_datefmt)
```

Comment:
Setting this here seemed reasonable considering that `self.datefmt` is used as early as [line 212](https://github.com/benoitc/gunicorn/pull/2379/files#diff-3f0ccdab4e1ca658f432ac7feb652ccaR212) to set up the error handler. However, is it more appropriate to have this override occur later on [line 232](https://github.com/benoitc/gunicorn/pull/2379/files#diff-3f0ccdab4e1ca658f432ac7feb652ccaR232)?

Reply:
Ah, I'm re-reading [the python logging docs](https://docs.python.org/3/library/logging.config.html#logging-config-dictschema) and see that the datefmt for each handler will need to be set from dicts in `cfg.logconfig_dict.formatters`